fake_graphs.py

```diff
@@ -77,9 +77,7 @@ def start_graph() -> nx.Graph:
 
     edges = [
         ('v:0:0', 'v:16:0', {'label': 'E', 'B': 1}),
-        ('v:16:0', 'v:16:16', {'label': 'E', 'B': 1}),
         ('v:16:16', 'v:0:16', {'label': 'E', 'B': 1}),
-        ('v:0:16', 'v:0:0', {'label': 'E', 'B': 1}),
```

Comment:
First noticed that this two edges were unnecessary. 
![image](https://github.com/user-attachments/assets/fb8d8b5a-2faf-4244-bc54-4f51cc88c80f)

They were here on image